Felienne/hedy
# Fix failing translations for hedy exceptions #1804

**Fix failing translations for all HedyExceptions**

* _Changes the translations of exceptions which did not have correct keywords and failed with KeyError when translated_
* _Adds automated tests that ensures no subclasses of HedyException fails because of wrong arguments. The tests are parametrized to automatically include newly added languages and exceptions._

**Fixes #1804 **

**How to test**

* To ensure the tests are passing, some of the already translated error messages had to be changed back to English. Is this ok? Maybe there is a nicer way to adapt the translation so that we do not completely lose it?
* Check how the tests are created. The idea of the tests is that they will be automatically created when a new language or exception is created but is the code easy to understand? Is the name of each individual test clear?
